include/deal.II/base/table.h

```diff
@@ -1549,6 +1549,18 @@ class Table<3, T> : public TableBase<3, T>
         InputIterator   entries,
         const bool      C_style_indexing = true);
 
+  /**
+   * Reinitialize the object. Passes down to the base class
+   * by converting the arguments to the data type requested by the base class.
```

Comment:
I'm coming late to this patch, but there is nothing to be converted in this function. Is this a copy-paste mistake, or am I just not understanding what you are referring to here?